pkgs/tools/misc/mermaid-filter/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, buildPackages
+, chromium
+}:
+
+buildNpmPackage rec {
+  pname = "mermaid-filter";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "raghur";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5MKiUeiqEeWicOIdqOJ22x3VswYKiK4RSxZRzJntO6M=";
+  };
+
+  npmDepsHash = "sha256-pnylo3dPgj7aD5czTWSV+uP5Cj8rVAsjZYoJ/WPRuuc=";
+
+  nativeBuildInputs = [ buildPackages.makeWrapper ];
+
+  prePatch = ''
```

Comment:
The `prePatch` is not often used.
It seems better to use `preBuild` or directly `env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD = 1`.